extension/commands/UserInputUtil.ts

```diff
@@ -1100,6 +1100,24 @@ export class UserInputUtil {
 
     }
 
+    public static async showOpsToolNodesQuickPickBox(prompt: string, nodes: FabricNode[], canPickMany: boolean): Promise<Array<IBlockchainQuickPickItem<FabricNode>> | IBlockchainQuickPickItem<FabricNode> | undefined> {
```

Comment:
can we change the name of this function as although you use it for ops tools its not specific to ops tools. I know there is a similar function for choosing nodes, so maybe rename them both, like `showNodesIn Environment` and `showNodes`?

Reply:
All done.